### ISSUE TITLE
ANSIBLE_PYTHON_INTERPRETER env variable not used?

### ISSUE THREAD

**Issue by sscherfke** [dw/mitogen#650]
When I run ansible *without* mitogen, I can set the remote Python interpreter via the envirnoment variable `ANSIBLE_PYTHON_INTERPRETER`:
```console
ANSIBLE_PYTHON_INTERPRETER=/usr/bin/python3 ansible-playbook deploy.yaml -vvv
```
The output confirms that, e.g., python3.6 is than used instead of python(2).

When I do the same with mitogen enabled, it will still use python(2). When I use `ansible_python_interpreter` from within the inventory file, it works, however.

**Comment by frankruegamer** [comment]
I can confirm this issue.

Also setting `interpreter_python = /usr/bin/python3` inside ansible.cfg will still result in python 2 being used.

This will also result in different behavior compared to vanilla Ansible, for example when using the `pip` module.

**Comment by bradh352** [comment]
duplicate of #597 ?

**Comment by s1113950** [comment]
https://github.com/dw/mitogen/pull/658 fixes this, and as #597 is closed I'm closing this as well. Please reopen if your issue persists!
